lockbox-ios/View/ItemListView.swift

```diff
@@ -91,6 +91,10 @@ extension ItemListView: ItemListViewProtocol {
         return self.tableView.rx.isScrollEnabled.asObserver()
     }
 
+    var pullToRefreshActive: AnyObserver<Bool> {
+        return self.tableView.refreshControl!.rx.isRefreshing.asObserver()
```

Comment:
Rather than force-casting, make the `AnyObserver<Bool>` optional and handle that case in the presenter.